custom-login/README.md

```diff
@@ -55,13 +55,16 @@ Now start the app server:
 npm start
 ```
 
+> **Note:** If you are on a windows machine, you might get an error related to `browserslist`.
+> It is a [known issue](https://github.com/angular/angular-cli/issues/5075) on windows due to one of the dependencies we use. To resolve the issue, search for `browserslist` and `browserslist.cmd` files in your node_modules. Delete the files and start the app server again.
```

Comment:
Not sure if this was only on the Angular sample, but my Windows 10 device isn't generating any of the 'browserslist.x` files.

Did you run into this while testing locally?

Reply:
I ran into it on a windows 2012 server VM. After I deleted it, I haven't run into it again.
I also tried on windows 2010 server VM but couldn't reproduce the error.
It might be possible that users never hit this issue. But in case they do, this might be a useful pointer.